addon/src/test-support/time-control.js

```diff
@@ -1,6 +1,7 @@
-import { clock, rAF } from 'ember-animated';
+import { clock, rAF } from '../-private/concurrency-helpers';
+import { getOrCreate } from '../-private/singleton';
 
-let origNow = clock.now;
+let origNow = getOrCreate('time-control', () => clock.now);
```

Comment:
@ef4 so this is the "trick" I've mentioned during Embroider Office Hour, which is a workaround for https://github.com/ef4/ember-auto-import/issues/503